stock_picking_return_restricted_qty/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Carlos Roca <carlos.roca@tecnativa.com>
```

Comment:
Write your name under a link to the Tecnativa website like it is here
https://github.com/OCA/web/blob/12.0/web_decimal_numpad_dot/readme/CONTRIBUTORS.rst